livedata/build.gradle

```diff
@@ -68,7 +68,7 @@ android {
 dependencies {
     implementation fileTree(dir: 'libs', include: ['*.jar'])
 
-    api "com.github.getstream:stream-chat-android-client:1.15.5"
+    api "com.github.getstream:stream-chat-android-client:1.15.6"
```

Comment:
This doesn't exist yet, so the pipeline will be broken for now. Can be tested locally by publishing the new version of the LLC from the appropriate branch and depending on that.